RentACar/RentACar/Options/FilePathsStorage.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace RentACar.Options
+{
+    public class FilePathsStorage
```

Comment:
minor: let's follow conventions by adding the `Options` suffix to this class name

Reply:
Oh yes! you mentioned that before! thanks!!